Add uninstall guide for Modern Client (#20)
kuba2k2/spicetify-beat-saber: README.md

7. `spicetify apply`
8. The app should now be available in your Spotify client.

> ### Uninstall - Modern clients (v1.2.x+)
> 
> 1. Run `spicetify config-dir` to open your Spicetify directory.
> 2. Go to `CustomApps` and remove `beatsaber`.
> 3. Run `spicetify config custom_apps beatsaber-` to remove it out of Spicetifys config.
> 4. `spicetify apply`
> 5. You should now have uninstalled the app :3
> 

Reviewer: ```suggestion
#### Uninstall - Modern clients (v1.2.x+)

1. Run `spicetify config-dir` to open your Spicetify directory.
2. Go to `CustomApps` and remove `beatsaber`.
3. Run `spicetify config custom_apps beatsaber-` to remove it out of Spicetify config.
4. `spicetify apply`
5. The app should now be uninstalled.
```
